api.swagger.yaml

```diff
@@ -1619,7 +1619,7 @@ paths:
         '200':
           description: The user reported this post as inappropriate.
         '400':
-          description: The UUID was malformed.
+          description: The UUID or request body JSON was malformed.
```

Comment:
On second thoughts - shouldn't it be 422 if the JSON was well-formed but didn't match the schema?

Reply:
updated